### ISSUE TITLE
Admin panel keeps logging me out.

### ISSUE THREAD

**Issue by Antypodish** [impresspages/ImpressPages#804]
Hi,

First of all, ImpressPages seams like great product.
I am learning it atm. 
However, it wasn't without issues.

I faced twice this problem, when just after logging in via admin panel, system just loges me out. I am unable to add new components, nor select anything from admin drop menu. Any attempts redirects me to my homepage, as just visitor. And I need to log in again. But no use.

Just to note, I use FTP, for file transfers, where I do certain modification on my machine.

First time, I realized, I have altered _header.php file, and system don't like <div> in certain places.
I just reloaded the original file and modified as required. I t worked and I could log in to admin panel.

However, second time (now), I did use some widgets for custom html. I suspect this is the issue. 
Reloading older files via FTP, seams do nothing. Switching to previous revision in admin panel, logs me out, same as any other task.

Any help? Any solutions? Any suggestions?

Best Regards

**Comment by jankus** [comment]
It depends what your modifications are. It could be an issue that you don't include JavaScript from the core. Then there's no way for admin tools to load. You may think that you are logged out when actually you're missing JavaScript.

Log in and try to go to `www.example.com/?aa=Design.index ` (your installation location with `?aa=Design.index` at the end). If admin loads, it means problem is with your theme. If it doesn't, it may be an issue with Sessions in your server.

By the way, did you see any warnings or issues during installation process?

**Comment by Antypodish** [comment]
That is superb fast response. Thx

With a ?aa=Design.index still the same issue.
It appears like logged in, but when dong anything, for example dragging new widged, I got js popoup alert message, with "undefided" text.

Same of firefox and chrome.

I am using x10hosting.com All installation seamed went smooth, no errors. And all worked up to the point I did some changes. 

Where is the publish button, there is a list of previous revisions of the web pages. I am trying to find on the server them to manually override files, as I can not select any version of that list. Where can I find directory location of revision files on the server?
I would expect them in "file" directory somewhere?

PS.

I just checked now, and it start working at sudden. At least for now ;)
Not sure, if "?aa=Design.index" has triggered something?

Many thx.

**Comment by jankus** [comment]
Something is definitely wrong either with installation (missing or corrupted files) or with your hosting.

Revisions are not about files. Revisions are entries in the database for each page you look at. When you click "save" or "publish" new revision is created.

**Comment by Antypodish** [comment]
Ok, seams I found the issue. Which is completely weird in my opinion.

I remember, that problem has occurred, after I  used edidt HTML widged.
Wat I was trying to do, to have wide bar, wider than main container.

Even most of admin panel was none functional, this widged was still editable.

Here is a code example:
```
<div style="position:relative; left:-50%; width:200%; background-color:red; background-image: url('img/header_bg.png'); background-repeat: repeat-x; background-size: contain;">
abc
</div>
```

**Nothing weird, until ...** 
So i had that logged out problem. Checked into the debug console. Nothing outstanding. Other than missing image. Simply, I didn't have "img/header_bg.png" directory nor the file. Decided to give a try, and add "img" folder to main directory and put missing png.

**And what?**
And voila. It works now.

Weird isn't it? Simple image issue.

I think that is good feedback for your team.

Thx for your time. 
I hope it will keep working ;)
